src/components/autocomplete/demoInsideDialog/dialog.tmpl.html

```diff
@@ -9,7 +10,7 @@
     </div>
   </md-toolbar>
 
-  <md-dialog-content>
+  <md-dialog-content ng-cloak>
```

Comment:
I think the `ng-cloak` is only useful on the `<md-dialog-content>` wrapper. All others are superfluous.